### PR TITLE
[IMPROVEMENT] Update README and add Installation file.

### PR DESCRIPTION
**Pull request by MatejMecka**
Add INSTALLATION file and update README

Please prefix your pull request with one of the following: **[FEATURE]** **[FIX]** **[IMPROVEMENT]**.

**In raising this pull request, I confirm the following (please check boxes):**

- [x] I have read and understood the [contributors guide](https://github.com/CCExtractor/ccextractor/blob/master/.github/CONTRIBUTING.md).
- [x] I have checked that another pull request for this purpose does not exist.
- [x] I have considered, and confirmed that this submission will be valuable to others.
- [x] I accept that this submission may not be used, and the pull request closed at the will of the maintainer.
- [x] I give this submission freely, and claim no ownership to its content.

**My familiarity with the project is as follows (check one):**

- [ ] I have never used CCExtractor.
- [ ] I have used CCExtractor just a couple of times.
- [ ] I absolutely love CCExtractor, but have not contributed previously.
- [x] I am an active contributor to CCExtractor.

---

Updated README and added an Installation file for offline users.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -29,227 +29,7 @@ More usage information can be found on our website:
...
-    cd ccextractor/package_creators
-    ./arch.sh
-    
-### Redhat Package Manager (rpm) based Linux Distributions
-
-*building installation package (.rpm)*
-
-    cd ccextractor/package_creators
-    ./rpm.sh
+To learn more about how to install CCExtractor for your platform check the INSTALLATION.MD file or visit the wiki and choose the operating system you want to do so or click [here](https://github.com/MatejMecka/ccextractor/wiki) to go there.
```
Comment (canihavesomecoffee) [inline README.md]: That link won't do. A wiki isn't good either. You can link to documents in the same repo, see the Sample platform for example.

File: INSTALLATION.MD [hunk 2]
```diff
@@ -0,0 +1,181 @@
+# Linux
```
Comment (canihavesomecoffee) [inline INSTALLATION.MD]: This lacks a good header, an overview to the available options and maybe an introduction.

File: INSTALLATION.MD [hunk 3]
```diff
@@ -0,0 +1,164 @@
+# Installation
+CCExtractor runs on all major platforms and getting started and using it is easy!
+
+You can compile it on most platforms using CMake using the `CMakeLists.txt` file stored in `ccextractor/src` autoconf and build scripts designed specifically for your platform.
```
Comment (canihavesomecoffee) [inline INSTALLATION.MD]: This sentence doesn't sound very right

File: INSTALLATION.MD [hunk 4]
```diff
@@ -0,0 +1,164 @@
...
+
+ For how to install using those two methods you can check the short guide for your platform.
+
+There are also precompiled binaries for you too use. You can get them from [CCExtractor’s](https://www.ccextractor.org/?id=public:general:downloads) website
+
+### Linux
```
Comment (canihavesomecoffee) [inline INSTALLATION.MD]: Why jump in 2 levels?

File: INSTALLATION.MD [hunk 5]
```diff
@@ -0,0 +1,164 @@
...
+### Fedora
+
+###### Installation
+In Order to install CCExtractor you need to have the following libraries installed: `gcc,tesseract-devel` Don't worry tesseract-devel comes default with leptonica
+
+You can install them using yum or paste this in your terminal
+```
+sudo yum install -y gcc
+sudo yum install -y tesseract-devel
+```### Setup
```
Comment (canihavesomecoffee) [inline INSTALLATION.MD]: This seems to be an error

File: INSTALLATION.MD [hunk 6]
```diff
@@ -0,0 +1,164 @@
...
+There are 3 ways to set up ccextractor depending on your choice.
+
+1. First and easiest way is to go in the linux folder and use the build script.
```
Comment (canihavesomecoffee) [inline INSTALLATION.MD]: Why not using headings with the names of the scripts?

File: INSTALLATION.MD [hunk 7]
```diff
@@ -0,0 +1,164 @@
...
+### Windows
+
+###### Installation
+Open the windows/ccextractor.sln file with Visual Studio (2015 at least), and build it.
+
+Configurations options are: `(Debug|Release)-Full`
+
+Configurations options include dependent libraries which are used for OCR.
+
+---
```
Comment (canihavesomecoffee) [inline INSTALLATION.MD]: What's up with these lines?

File: README.md [hunk 8]
```diff
@@ -29,227 +29,7 @@ More usage information can be found on our website:
...
-    cd ccextractor/package_creators
-    ./arch.sh
-    
-### Redhat Package Manager (rpm) based Linux Distributions
-
-*building installation package (.rpm)*
-
-    cd ccextractor/package_creators
-    ./rpm.sh
+To learn more about how to install CCExtractor for your platform check the [installation guide](INSTALLATION.MD).
```
Comment (canihavesomecoffee) [inline README.md]: I'd put this on top of this section, and remove the precompiled binaries section as that's also mentioned in the installation guide.

File: INSTALLATION.MD (hunk 7 above)
Comment (MatejMecka) [reply INSTALLATION.MD]: line break or separator

**Pull request merged**